rdserial/dps/tool.py

```diff
@@ -161,6 +162,41 @@ def send_commands(self):
                 register_base, register_commands_opt[register_base], unit=self.args.modbus_unit,
             )
 
+    def send_command(self, command, argument=None):
```

Comment:
I need to fix this up to get the registers from the device class not hard coded like this